modules/database/src/Database.ts

```diff
@@ -816,6 +816,10 @@ export default class DatabaseModule extends ManagedModule<void> {
         delete (syncSchema as any).fieldHash;
         await this._activeAdapter.createSchemaFromAdapter(syncSchema, false, false, true);
       });
+      this.grpcSdk.bus?.subscribe('database:create:view', async viewStr => {
```

Comment:
Should technically be `database:view:create` according to our convention (that also seems to be broken around `customSchema` events).